index.html

```diff
@@ -559,6 +559,22 @@ <h3>Use Cases and Requirements</h3>
 or <a>verifiers</a>, because the conformance of ecosystem roles are highly
 application, use case, and market vertical specific.
         </p>
+
+        <p>
+While digital proofs, a subset of which are digital signatures, are required
+to ensure the protection of a <a>verifiable credential</a>, this specification
+does not standarize on any single digital signature format. At the time of
+publication, the Working Group included members that had implemented using
+at least three proof mechanisms. These implementations utilized
```

Comment:
"...proof Mechanisms: JSON Web Tokens [ref]; Linked Data Signatures [ref]..." ?

Reply:
Fixed in 276007376a0082128439f109c32779c50f740656.